packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -240,7 +240,7 @@ export class DashboardModel {
         const cteTableName = 'cte';
         const dashboardsQuery = this.database
             .with(cteTableName, (queryBuilder) => {
-                queryBuilder
+                void queryBuilder
```

Comment:
knex query builder returns a promise every time you "mutate" the query.. so we can safely mark them with `void`. there's bunch of them in the upcoming files!